packages/tools/src/utilities/getModalityUnit.ts

```diff
@@ -0,0 +1,11 @@
+function getModalityUnit(modality: string): string {
+  if (modality === 'CT') {
+    return 'HU';
+  } else if (modality === 'PT') {
```

Comment:
We can do both checks here for 'PT'. The check that sees if the data is preScaled. I guess this function can also get the `imaegId` or `volumeId` (can be extracted from `targetId`), and checks if the volume/image is prescaled. I have similar operation here (https://github.com/cornerstonejs/cornerstone3D-beta/blob/7c311f77f0532372ae82b6be2027bcd25925fa0d/packages/tools/src/tools/WindowLevelTool.ts#L71-L77) Let me know what you think

Reply:
So right now the StackViewport gives back the `preScale` flag, but we don't have that on the VolumeViewport since it can be either volumes (in case of fusion for example).

Reply:
Thanks for the link to your code. I added it to a helper function to check for pre-scaling and pass the flag to the modality unit function.